weblate/utils/requirements.py

```diff
@@ -83,9 +84,6 @@
     "boto3",
     "zeep",
     "aeidon",
-    "iniparse",
-    "mysqlclient",
-    "pyparsing",
```

Comment:
The removal looks wrong.

Reply:
I used pycharm code refactor tool. It modified it somehow. It's fixed now.